SS Rando Logic - Glitchless Requirements.yaml

```
@@ -37,7 +37,19 @@ Hook Beetle:
   Progressive Beetle x2
 
 Quick Beetle:
-  Hook Beetle & Can Access Deep Woods & Can Activate Mines First Timeshift Stone & Can Defeat Bokoblins
+  (Hook Beetle & Emerald Tablet & Amber Tablet & Can Defeat Bokoblins & Option "gondo-upgrades" Disabled) | (Progressive Beetle x3 & Option "gondo-upgrades" Enabled)
```

```suggestion
  (Hook Beetle & Emerald Tablet & Amber Tablet & Can Defeat Bokoblins & Option "gondo-upgrades" Disabled & Upgrade Hook Beetle Trick) | Progressive Beetle x3
```

There are only 2 beetles in the pool without the option enabled. In the future plandomizers might be able to alter the item pool, so even without the option getting 3 beetles could be possible.